lib/CodeGen/CGException.cpp

```diff
@@ -1737,7 +1738,8 @@ void CodeGenFunction::EmitSEHExceptionCodeSave(CodeGenFunction &ParentCGF,
     // pointer is stored in the second field. So, GEP 20 bytes backwards and
     // load the pointer.
     SEHInfo = Builder.CreateConstInBoundsGEP1_32(Int8Ty, EntryFP, -20);
-    SEHInfo = Builder.CreateBitCast(SEHInfo, Int8PtrTy->getPointerTo());
+    unsigned DefaultAS = CGM.getTargetCodeGenInfo().getDefaultAS();
+    SEHInfo = Builder.CreateBitCast(SEHInfo, Int8PtrTy->getPointerTo(DefaultAS));
```

Comment:
I'm not sure how SEH would work in a CHERI model, but given that we're not supporting Windows (at least, not any time soon) then it probably doesn't matter...